openedx/core/lib/logsettings.py

```diff
@@ -126,6 +128,10 @@ def log_python_warnings():
     warnings.filterwarnings('ignore', 'Setting _field_data is deprecated')
     warnings.filterwarnings('ignore', 'Setting _field_data via the constructor is deprecated')
     warnings.filterwarnings('ignore', '.*unclosed.*', category=ResourceWarning)
+    warnings.filterwarnings('ignore', '.*You can remove default_app_config.*', RemovedInDjango41Warning)
+    warnings.filterwarnings('ignore', '.*Instead access HTTPResponse.headers directly.*')
```

Comment:
It would be nice to add a more specific [warning category](https://docs.python.org/3/library/warnings.html#warnings.filterwarnings).

Reply:
agree